service/azureconfig/v1/resource/resourcegroup/resource.go

```diff
@@ -19,34 +19,38 @@ const (
 	// Name is the identifier of the resource.
 	Name = "resourcegroupv1"
 
-	clusterIDTag  = "ClusterID"
-	customerIDTag = "CustomerID"
 	deleteTimeout = 30 * time.Minute
 	managedBy     = "azure-operator"
 )
 
 type Config struct {
-	AzureConfig client.AzureConfig
-	Logger      micrologger.Logger
+	AzureConfig      client.AzureConfig
+	InstallationName string
+	Logger           micrologger.Logger
```

Comment:
Nitpick InstallationName is more of a setting than dep.

Reply:
Good catch, done.